index.html

```diff
@@ -2,6 +2,8 @@
 <html>
```

Comment:
Don't see you adding a semantic HTML tag, per Question 2. That would be something like `<main>`.